engine/installation/binaries.md

```diff
@@ -4,243 +4,199 @@ keywords: binaries, installation, docker, documentation, linux
+The macOS binary includes the Docker client only. It does not include the
+`dockerd` daemon.
 
-You can manually start the Engine in daemon mode using:
+1.  Download the static binary archive. You can download either the latest
+    release binaries or a specific version. To find the download link, see the
+    [release notes](https://github.com/docker/docker/releases) for the version
+    of Docker you want to install. You can choose a `tar.gz` archive or
+    `zip` archive.
 
-```bash
-$ sudo dockerd &
-```
+2.  Extract the archive using `tar` or `unzip`, depending on the format you
+    downloaded. The `docker` binary is extracted.
 
-The GitHub repository provides samples of init-scripts you can use to control
-the daemon through a process manager, such as upstart or systemd. You can find
-these scripts in the <a href="https://github.com/docker/docker/tree/master/contrib/init">
-contrib directory</a>.
+    ```bash
+    $ tar xzvf /path/to/<FILE>.tar.gz
+    ```
 
-For additional information about running the Engine in daemon mode, refer to
-the [daemon command](../reference/commandline/dockerd.md) in the Engine command
-line reference.
+    ```bash
+    $ unzip /path/to/<FILE>.zip
+    ```
+3.  **Optional**: Move the binaries to a directory on your executable path, such
+    as `/usr/local/bin/`. If you skip this step, you must provide the path to the
+    executable when you invoke `docker` or `dockerd` commands.
 
-### Get the macOS binary
+    ```bash
+    $ sudo cp docker/docker /usr/local/bin/
+    ```
 
-The macOS binary is only a client. You cannot use it to run the `docker`
-daemon. To download the latest version for macOS, use the following URLs:
+4.  To connect the `docker` client to a remote host, either set the
+    `DOCKER_HOST` environment variable to the hostname or IP address of the
+    Docker daemon or add the `-H <hostname|IP>` option to each `docker` command.
 
-    https://get.docker.com/builds/Darwin/x86_64/docker-latest.tgz
+5.  Verify that Docker is installed correctly by running the `hello-world`
+    image.
 
-To download a specific version for macOS, use the
-following URL pattern:
+    ```bash
+    $ sudo docker -H <hostname> run hello-world
+    ```
 
-    https://get.docker.com/builds/Darwin/x86_64/docker-<version>.tgz
+    This command downloads a test image and runs it in a container. When the
+    container runs, it prints an informational message and exits.
 
-For example:
 
-    https://get.docker.com/builds/Darwin/x86_64/docker-1.11.0.tgz
+## Install server and binaries on Windows
 
-You can extract the downloaded archive either by double-clicking the downloaded
-`.tgz` or on the command line, using `tar -xvzf docker-1.11.0.tgz`. The client
-binary can be executed from any location on your filesystem.
+You can install Docker from binaries on Windows Server 2016 or Windows 10.
 
+- To install both client and server binaries, download the 64-bit binary. The
+  archive includes `x86.64` in the file name.
```

Comment:
it's x86_64 and it's in the path, not the filename (that goes for the client below too)